appvm-scripts/etc/qubes/post-install.d/20-qubes-guivm-gui-agent.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+if rpm -q xorg-x11-server-Xephyr > /dev/null; then
```

Comment:
Check binary existence, not the package. Otherwise it won't work on other distributions.